fontend/app/staff/page.tsx

```
@@ -0,0 +1,444 @@
+}
+
+interface UserStats {
+    total: number;
+    active: number;
+    inactive: number;
+    adminWeb: number;
+    staffWeb: number;
+    defaultUsers: number;
+}
+
+interface Users {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Users'. Consider to delete it or rename it.

'Users' is defined here:


(lint/suspicious/noRedeclare)